### PR TITLE
Adding credits page to list our subscribers & open source projects

### PR DESCRIPTION
**Pull request by clone1018**
image

### CODE REVIEW

**Comment by clone1018** [comment]
@haydenmc can you check out the deps for the FTL projects? Stole it from the masons.

File: lib/glimesh_web/templates/about/credits.html.eex [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                style="width: 80% !important;" class="img-fluid mb-2 img-avatar">
+        </div>
+        <% end %>
+    </div>
+
+    <% end %>
+</div>
+
+<h2><%= gettext("Open Source Libraries") %></h2>
+<p><%= gettext("We use numerous open source projects to power Glimesh, and without them this project would never have been possible. This is not a comprehensive list unfortunately. You can find a full list by browsing our core repositories:") %>
```
Comment (mja00) [inline lib/glimesh_web/templates/about/credits.html.eex:41]:
```suggestion
<p><%= gettext("We use numerous open source projects to power Glimesh, and without them this project would never have been possible. This is not a comprehensive list, unfortunately. You can find a full list by browsing our core repositories:") %>
```

Just a small grammatical thing :)

**Review by mja00 — COMMENTED** [review]
Looking good :)

**Review by haydenmc — COMMENTED** [review]
libcrypto is considered part of openssl, it might make sense to only mention openssl here

File: lib/glimesh/credits.ex [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      {"glib-2.0",
+       "GLib provides the core application building blocks for libraries and applications written in C."},
+      {"libsrtp2",
+       "Implementation of the Secure Real-time Transport Protocol (SRTP), the Universal Security Transform (UST), and a supporting cryptographic kernel. "},
+      {"jansson", "Jansson is a C library for encoding, decoding and manipulating JSON data."},
+      {"libssl",
+       "OpenSSL is a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols."},
+      {"libcrypto", "General-purpose cryptography library"},
+      {"libavcodec",
+       "Generic encoding/decoding framework and contains multiple decoders and encoders for audio, video and subtitle streams, and several bitstream filters."},
```
Comment (haydenmc) [inline lib/glimesh/credits.ex:36]: Might be worth mentioning that this is provided by ffmpeg folks

File: lib/glimesh/credits.ex [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      {"spdlog", "Very fast, header-only/compiled, C++ logging library."}
+    ]
```
Comment (haydenmc) [inline lib/glimesh/credits.ex:38]: For Orchestrator we're also using Catch2 test framework.

And for http we're using https://github.com/yhirose/cpp-httplib

**Pull request merged**